internal/receiver/scriptedinputsreceiver/scripts/cpu.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/sh
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is this the existing script license?

Reply:
These scripts are taken from Splunk Addon for Unix and Linux, below is the license from one of the files:
# SPDX-FileCopyrightText: 2021 Splunk, Inc. <sales@splunk.com>
# SPDX-License-Identifier: Apache-2.0